src/client/pages/Forever/components/RedeemCode.tsx

```diff
@@ -185,7 +185,6 @@ export const RedeemCode: React.FC<RedeemCodeProps> = ({
                   }
                   name="code"
                   type="text"
-                  autoComplete="off"
```

Comment:
Why do we change this?

Reply:
It is hard for me to see a scenario where your browser already knows the redeemed code for this so it actually uses the autoComplete option.

Reply:
Hmm.. I think the browser might decide to suggest any number of irrelevant things so I would keep "autoComplete" turned off.